src/azure-cli/azure/cli/command_modules/vm/generated/_help.py

```diff
@@ -12,6 +12,11 @@
 from knack.help_files import helps
 
 
+helps['vm'] = '''
+    type: group
+    short-summary: Manage Compute
+'''
+
```

Comment:
Need to be updated

Reply:
Generated help info could be covered by manual help info